packages/emitter-framework/src/python/components/protocol-declaration/protocol-declaration.tsx

```diff
@@ -0,0 +1,77 @@
+import { useTsp } from "#core/context/tsp-context.js";
+import { typingModule } from "#python/builtins.js";
+import { TypeExpression } from "#python/components/type-expression/type-expression.js";
+import { reportPythonDiagnostic } from "#python/lib.js";
+import { declarationRefkeys } from "#python/utils/refkey.js";
+import { mapJoin } from "@alloy-js/core";
+import * as py from "@alloy-js/python";
+import type { Interface, Operation, Type } from "@typespec/compiler";
+import { CallableParameters } from "./callable-parameters.js";
+
+export function Ellipsis() {
```

Comment:
I don't think a component is needed for this, shorter to just type "..." where needed.